docsearch/hbs-helpers/versioned.js

```diff
@@ -1,3 +1,3 @@
-const versioned = (components) => components.filter(({ latest }) => latest !== 'master')
+const versioned = (components) => components.filter(({ latest }) => (latest || 'master') != 'master')
```

Comment:
Rather than doing a catch-all for latest here, it would be better to check for the component with the exception by name. That way, nothing surprising happens when a different component shows up.

Reply:
Ah? I thought you've mentioned that `master` and `~` were semantically equivalent, with the latter simply being the preferred modern option?

If another component ends up having a `~` version, wouldn't that also need to be treated as versionless? (e.g. lose the `/current/` path etc.)

Reply:
They are semantically equivalent, but only because Antora adds special treatment to the value of "master". The value "master" is still the version value if set that way.

Having a closer look at the code, I see that what it is doing is coercing an empty/undefined value to "master" for the purpose of comparison. I suppose this is reasonable after all.

Reply:
Discussing with @leeming it's definitely a bit unclear!
Will rewrite to something like `((latest === 'master') or (latest === ''))`